portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,39 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body,
+html {
+    margin: 0;
+    padding: 0;
+}
+.jumbotron {
+    align-content: center;
```

Comment:
Try align-items instead, I think that might be what you're looking for

Reply:
That was it, along with a small margin change. How would I update this commit with this change, or do I just make a new one?

Reply:
Updating the branch should update the PR accordingly I think